slides/swarm/firstservice.md

```diff
@@ -379,6 +379,14 @@ class: btw-labels
 
   - labels are detected automatically and dynamically update the configuration
 
+- Two common open source "reverse proxy" options
+
+  - [Traefik](https://traefik.io/) - uses NGINX, popular, requires proxy be on managers, 
```

Comment:
Are you sure that Traefik uses NGINX? As far as I know, it's entirely written in Go...

Reply:
wow 🤦‍♂️ I always thought it was a nginx wrapper... clearly I need to dig deeper. Thanks will update this branch.